site/docs/iceberg/trino.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Nessie + Iceberg + Trino"
+---
+
+# Trino via Iceberg
+
+Trino comes with Iceberg and Nessie catalog clients installed by default. 
+You can deploy Trino using any of the methods mentioned in [this document](https://trino.io/docs/current/installation.html) and to become familiar with the Iceberg connector, refer [this document](https://trino.io/docs/current/connector/iceberg.html?highlight=iceberg+connector).
```

Comment:
maybe `Trino can be deployed for use with Nessie by any of the methods mentioned in the [Trino Installation Guide](https://trino.io/docs/current/installation.html). To access Iceberg tables, the Iceberg connector is required. Refer to the [Trino Connector Documentation](*link*) for more information`?

Reply:
done